client/scripts/BUILDINGOPTIONSPOPUP.as

```diff
@@ -198,7 +198,8 @@ package
                   "r2":0,
                   "r3":0,
                   "r4":0,
-                  "r5":0
+                  "r5":0,
+                  "time":new SecNum(_loc3_.time.Get())
```

Comment:
@AMXBus - can you explain why we add a new time property onto this? My question around this is, if the original game didn't get the time from here, then where did it get from? And why do we give it a time?

Reply:
During investigation on where all the time properties get called there are a few areas where if the time property was not ever added or got overwritten it would have a resolution on it's own. When I implemented SecNum however in all those calls many submenus would just refuse to work. Even some that get filled at runtime would stop the game with an exception.

This was most likely me searching over all mentions and usages of the time property of buildings get used and covering all bases.